rules/S799/cfamily/metadata.json

```
@@ -3,7 +3,8 @@
   "type": "CODE_SMELL",
   "code": {
     "impacts": {
-      "MAINTAINABILITY": "MEDIUM"
+      "MAINTAINABILITY": "MEDIUM",
+      "SECURITY": "MEDIUM"
```

```suggestion
      "MAINTAINABILITY": "MEDIUM"
```

I can't see any security impact?